src/renderer/families/tron/Votes/index.js

```diff
@@ -61,6 +62,24 @@ const useNewVotingDate = () => {
   return nextVotingDate;
 };
 
+// @TODO move this to common
+const getNextRewardDate = (account: Account) => {
+  const { operations } = account;
+  const lastRewardOp = operations.find(({ type }) => type === "REWARD");
+
+  if (lastRewardOp) {
+    const { date } = lastRewardOp;
+    if (date) {
+      // add 24hours
+      const nextDate = new Date(date.valueOf());
+      nextDate.setDate(date.getDate() + 1);
```

Comment:
i think this does not always works (last day of month?)

it sounds like you can replace all these by `date.getTime() + 24*60*60*1000 > Date.now()` ?